auth/tests/base.py

```diff
@@ -0,0 +1,33 @@
+from StringIO import StringIO
+
+from django.conf import settings
+from django.test import TestCase
+
+import requests
+from mock import patch
+
+
+class Proxy(TestCase):
```

Comment:
You had to call this Proxy, huh? ;)

It looks generic enough to me that you could just call it ``TestCase`` or ``BaseTest``.